test/Microsoft.Azure.WebJobs.Extensions.Kafka.LangEndToEndTests/command/type/CommandType.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Azure.WebJobs.Extensions.Kafka.LangEndToEndTests.command.type
+{
+    /* Collection of supported command types
+    */
+    public enum CommandType
```

Comment:
Are we using it anywhere , else we can remove.

Reply:
Removed.